0000-template.md

```diff
@@ -1,3 +1,14 @@
+<!---
+Copyright <fill in year> The Rust Project Developers. See the COPYRIGHT
```

Comment:
@eddyb Yes, it is. I had a conversation about that with @brson a while back when a new such instance was added, and I *thought* some of them were addressed at that time (by switching to a notice that said something about copyrights being retained by contributors). But we need to address the rest of those notices.